### PR TITLE
Fixed typesetting issue in usfmanus.cls to work for Latexmk

### PR DESCRIPTION
**Pull request by anwesht**
Fix is based on: https://tex.stackexchange.com/questions/102927/undefined-control-sequence-with-color-package-in-custom-latex-document-class

Cleaned up main.tex a bit as well.

### CODE REVIEW

File: Latex/usfmanus.cls [hunk 1]
```diff
@@ -872,7 +872,9 @@ University of South Florida Manuscript Class]
   \newpage
 }
 
-\AtBeginDocument{
+% \AtBeginDocument{
+\RequirePackage{etoolbox}
```
Comment (jailby) [inline Latex/usfmanus.cls]: There is already a `\RequirePackage{}` [here](https://github.com/jailby/dissertation_template/blob/master/Latex/usfmanus.cls#L8)

Does it still compile for you if you do `\RequirePackage{setspace,etoolbox}`

Comment (anwesht) [reply Latex/usfmanus.cls]: Yes can be moved there.

Comment (jailby) [reply Latex/usfmanus.cls]: Do you mind making the modification and committing it? To keep the class simpler

Comment (anwesht) [reply Latex/usfmanus.cls]: Sure.

**Review by jailby — APPROVED** [review]

**Pull request merged**